### PR TITLE
bpo-33303: ElementTree Comment text isn't escaped

### PR DESCRIPTION
**Pull request by websurfer5**
properly escape XML comment text, plus test

https://bugs.python.org/issue33303

### CODE REVIEW

File: Lib/xml/etree/ElementTree.py [hunk 1]
```diff
@@ -897,7 +897,7 @@ def _serialize_xml(write, elem, qnames, namespaces,
     tag = elem.tag
     text = elem.text
     if tag is Comment:
-        write("<!--%s-->" % text)
+        write("<!--%s-->" % _escape_cdata(text))
     elif tag is ProcessingInstruction:
         write("<?%s?>" % text)
```
Comment (scoder) [inline Lib/xml/etree/ElementTree.py:902]: Please also include the text of processing instructions.

File: Misc/NEWS.d/next/Library/2019-05-09-22-36-41.bpo-33303.B7_POx.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+properly escape XML comment text
```
Comment (scoder) [inline Misc/NEWS.d/next/Library/2019-05-09-22-36-41.bpo-33303.B7_POx.rst]: This is meant for humans to read, so please write a complete sentence, with a dot at the end.

File: Lib/xml/etree/ElementTree.py (hunk 1 above)
Comment (websurfer5) [reply Lib/xml/etree/ElementTree.py:902]: It now escapes "?>" with "&gt;>".

File: Misc/NEWS.d/next/Library/2019-05-09-22-36-41.bpo-33303.B7_POx.rst (hunk 2 above)
Comment (websurfer5) [reply Misc/NEWS.d/next/Library/2019-05-09-22-36-41.bpo-33303.B7_POx.rst]: Done

File: Lib/xml/etree/ElementTree.py (hunk 1 above)
Comment (scoder) [reply Lib/xml/etree/ElementTree.py:902]: Sorry for this, I was wrong. PIs are not escaped. Please revert the commit where you added it.

**Comment by scoder** [comment]
Also, since this is a backwards incompatible change, this also needs to be mentioned in the "What's new" document: https://github.com/python/cpython/blob/master/Doc/whatsnew/3.8.rst#xml

**Comment by scoder** [comment]
Thank you for your work and a perfect PR. I'm really sorry that I have to reject this feature.

**Pull request closed without merging**